Product/DBScripts/nhincdb/nhincdb.sql

```diff
@@ -631,6 +635,6 @@ UPDATE domain SET postmasterAddressId = 2 WHERE id = 1;
 
 USE eventdb;
 LOCK TABLES event WRITE;
-ALTER TABLE event MODIFY COLUMN eventTime timestamp null;
+ALTER TABLE event MODIFY COLUMN eventTime timestamp not null;

```

Comment:
Can these eventdb statements be removed altogether?  The timestamp type is not null by default.